src/test/java/au/org/democracydevelopers/raireservice/controller/GetAssertionsCSVAPITests.java

```diff
@@ -101,33 +102,32 @@ public void testValidRequestWithNoAssertions() {
     assertTrue(response.getStatusCode().is2xxSuccessful());
     String output = response.getBody();
```

Comment:
Add "assertNotNull(output);" to avoid the warning about the NullPointerException.